### ISSUE TITLE
accuracy = 0.000 after Epoch 132/10000

### ISSUE THREAD

**Issue by GunAndFun** [ilovin/lstm_ctc_ocr#2]
5-7 19:8:33 Epoch 130/10000, accuracy = 0.000,train_cost = 21.917, lastbatch_err = 0.983, time = 110.495
('batch', 99, ': time', 0.23030900955200195)
('batch', 199, ': time', 0.3604588508605957)
seq   0: origin: [42, 29, 32, 12, 36, 20] decoded:[40]
seq   1: origin: [30, 45, 45, 5, 54] decoded:[22, 19]
seq   2: origin: [62, 40, 57, 7, 58, 54] decoded:[46]
seq   3: origin: [48, 14, 3, 19] decoded:[19]
seq   4: origin: [17, 32, 3, 59, 29, 55] decoded:[40]
seq   5: origin: [20, 14, 55, 15] decoded:[47]
seq   6: origin: [10, 42, 29, 37, 26, 34] decoded:[46]
seq   7: origin: [22, 48, 36, 41, 26, 55] decoded:[57]
seq   8: origin: [58, 6, 27, 11, 17, 1] decoded:[39]
seq   9: origin: [52, 48, 57, 29, 14] decoded:[46]
5-7 19:11:8 Epoch 131/10000, accuracy = 0.000,train_cost = 21.915, lastbatch_err = 0.983, time = 154.524
('batch', 99, ': time', 0.36012792587280273)
('batch', 199, ': time', 0.3593289852142334)
seq   0: origin: [42, 29, 32, 12, 36, 20] decoded:[40]
seq   1: origin: [30, 45, 45, 5, 54] decoded:[13, 19]
seq   2: origin: [62, 40, 57, 7, 58, 54] decoded:[46]
seq   3: origin: [48, 14, 3, 19] decoded:[19]
seq   4: origin: [17, 32, 3, 59, 29, 55] decoded:[40]
seq   5: origin: [20, 14, 55, 15] decoded:[33]
seq   6: origin: [10, 42, 29, 37, 26, 34] decoded:[46]
seq   7: origin: [22, 48, 36, 41, 26, 55] decoded:[40]
seq   8: origin: [58, 6, 27, 11, 17, 1] decoded:[40]
seq   9: origin: [52, 48, 57, 29, 14] decoded:[46]
5-7 19:13:53 Epoch 132/10000, accuracy = 0.000,train_cost = 21.904, lastbatch_err = 0.984, time = 164.831
('batch', 99, ': time', 0.4470250606536865)
('batch', 199, ': time', 0.36761021614074707)
seq   0: origin: [42, 29, 32, 12, 36, 20] decoded:[40]
seq   1: origin: [30, 45, 45, 5, 54] decoded:[13, 40]
seq   2: origin: [62, 40, 57, 7, 58, 54] decoded:[40]
seq   3: origin: [48, 14, 3, 19] decoded:[40, 19]
seq   4: origin: [17, 32, 3, 59, 29, 55] decoded:[40]
seq   5: origin: [20, 14, 55, 15] decoded:[40]
seq   6: origin: [10, 42, 29, 37, 26, 34] decoded:[46]
seq   7: origin: [22, 48, 36, 41, 26, 55] decoded:[40]

**Comment by GunAndFun** [comment]
I just git clone your code and run on a GPU M40, the result a quite different from your blog's description ,where accuracy =0.730 after 24 epochs. Is there something wrong I did?

**Comment by ilovin** [comment]
I commented the learning rate decay code.
To get better result, you shall uncomment it, and try to adjust the parameters (such as decay rate and decay steps)
Besides, more training data is needed or try to use on-the-fly training
actually, there is already a similar issue.
https://github.com/ilovin/lstm_ctc_ocr/issues/1

**Comment by lmolhw5252** [comment]
我和他的问题一样，在多个epoch后，准确率还是0，这个咋解决？

**Comment by ilovin** [comment]
the same issue, see
https://github.com/ilovin/lstm_ctc_ocr/issues/1
for more detail

**Comment by duwei0203** [comment]
你好，train=128000.val=1000，learning_rate=1e（-3），经过十几个epoch之后acc依然为零，在主页我看到这句话decay step & decay rate (notice that, uncomment the learing rate decay part of code if it is commented)，我不清楚是否有需要取消注释的地方我没有取消，想问你能不能给出具体位置，谢谢！

**Comment by ilovin** [comment]
在self.optimizer = 哪里有一个learning_rate = 改称self.learning_rate
在standard中我以及默认改掉了
